lms/djangoapps/bulk_email/models.py

```diff
@@ -117,6 +121,13 @@ def get_users(self, course_id, user_id=None):
             courseenrollment__is_active=True
         )
         enrollment_qset = User.objects.filter(enrollment_query)
+
+        # filter out learners from the message who are no longer active in the course-run based on last login
+        last_login_eligibility_period = getattr(settings, 'BULK_COURSE_EMAIL_LAST_LOGIN_ELIGIBILITY_PERIOD', None)
```

Comment:
if `BULK_COURSE_EMAIL_LAST_LOGIN_ELIGIBILITY_PERIOD` is in `common.py` as `None` we don't need this `getattr` check. We can just `settings.BULK_COURSE_EMAIL_LAST_LOGIN_ELIGIBILITY_PERIOD` directly and it'll either be `None` (not overridden) or the overridden value.